google-account-plugin/src/com/google/gct/idea/AccountPluginInitializationComponent.java

```diff
@@ -63,13 +53,13 @@ void enableFeedbackUtil() {
    * and is therefore not subscribed and listening.
    */
   @VisibleForTesting
-  void enableUsageTracking() {
+  void maybeEnableUsageTracking() {
     UsageTrackerManager usageTrackerManager = UsageTrackerManager.getInstance();
-    NotificationsManager.getNotificationsManager();
-    NotificationsConfiguration.getNotificationsConfiguration().register(
-        TrackerMessageBundle.message("notification.group.display.id"),
-        NotificationDisplayType.STICKY_BALLOON);
     if (usageTrackerManager.isUsageTrackingAvailable() && !usageTrackerManager.hasUserRecordedTrackingPreference()) {
+      NotificationsManager.getNotificationsManager();
+      NotificationsConfiguration.getNotificationsConfiguration().register(
+          TrackerMessageBundle.message("notification.group.display.id"),
```

Comment:
As far as I have seen it in the code it is never displayed. It is used for some internal grouping of the notifications.